pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -543,6 +543,7 @@ def test_tda_add_sub_index(self):
         expected = tdi - tdi
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.xfail(reason="GH38630", strict=False)
```

Comment:
is there a reason you are passing strict? we default to true - eg if these are fixed we want the tests to fail (as a hint to remove the xfail)

Reply:
Problem I ran into is that some parameterizations actually pass, so `strict=True` fails for these. Couldn't figure out a way to `xfail` only the failing combinations because the parameterizations are complex (2 defined elsewhere in fixtures, 1 uses multiple calls to `pytest.mark.parametrize`).

Reply:
ok that's fine

ping on green

Reply:
Seeing 2 failures where `pandas/tests/io/parser/test_common.py::test_chunks_have_consistent_numerical_type[python] `gives unexpected `ResourceWarning`. 

Do you know if this warning occurs consistently? Should something in the test be modified to handle a potential `ResourceWarning`? Or just another xfail case?

Reply:
we ar trying to track these cases down as something is leaking 

if they r causing actual failures then ok to xfail (and list these in the associated issue with checkboxes)